dataland-frontend/src/assets/content.json

```diff
@@ -0,0 +1,376 @@
+                        {
+                            "icon": "/static/icons/Area.svg",
+                            "title": "5",
+                            "text": "Frameworks"
+                        }
+                    ]
+                },
+                {
+                    "title": "Join a campaign",
+                    "text": [
+                        "Join a campaign to",
+                        " to get the missing ",
+                        "data.",
+                        "The Proof of Concepts are a simple way for you to get better data coverage and improved data quality, while testing Dataland."
+                    ],
+                    "cards": [
+                        {
+                            "title": "Pathway to Paris",
+                            "text": "Explanatory text goes here",
+                            "date": "14th of August"
+                        },
+                        {
+                            "title": "LksG",
+                            "text": "Explanatory text goes here"
+                        },
+                        {
+                            "title": "EU Taxonomy",
+                            "text": "Explanatory text goes here"
+                        },
+                        {
+                            "title": "SFDR",
+                            "text": "Explanatory text goes here"
+                        }
+                    ]
+                },
+                {
+                    "title": "Get in touch",
+                    "text": [
+                        "Get in touch"
+                    ],
+                    "cards": [
+                        {
+                            "icon": "Email",
+                            "title": "Your message",
+                            "text": "Get in touch",
+                            "date": "By clicking Get in Touch,  you agree to our Terms of use and that you have read our Privacy Policy."
+                        },
+                        {
+                            "icon": "/static/images/927A74151.jpg",
+                            "title": "Erik Breen,",
+                            "text": "Managing Director of Dataland"
+                        },
+                        {
+                            "icon": "erik.breen@dataland.com",
+                            "title": "I am interested in Dataland’s campaigns 📣 ",
+                            "text": "Dear Erik,\n \nI am sick and tired of data gaps and bad data quality. Therefore, I am very much interested in the SFDR Campaign.\n \nPlease keep me updated.\n \nBest regards, "
+                        },
+                        {
+                            "icon": "erik.breen@dataland.com",
+                            "title": "Eliminate Data Gaps: Your Path to High-Quality Data Starts Here!",
+                            "text": "Dear Erik,\n \nI am interested in Dataland.\n \nBest regards, "
```

Comment:
I think this is not used. At least in `TheGetInTouch`

Reply:
It's used in the EmailButton component. In the next version there will be a contact form instead and this fields will be gone.